web/apps/labelstudio/src/components/Form/Elements/Label/Label.jsx

```diff
@@ -1,7 +1,7 @@
 import { createElement } from "react";
 import { cn } from "../../../../utils/bem";
 import "./Label.scss";
-
+// @deprecated - needs to be replaced with @humansignal/ui Label - visualizes differently currently
```

Comment:
```suggestion
/** @deprecated - needs to be replaced with @humansignal/ui Label - visualizes differently currently */
```
it's correctly parsed only in multiline comments